tests/TestClasses/ConvertersTests/PropertyTestConverter.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.ComponentModel;
+using System.Globalization;
+
+namespace XamlParserTests
+{
+    public class PropertyTestConverter : TypeConverter
+    {
+        public static event Action<CultureInfo, ITypeDescriptorContext> ConvertFromEventRequiredAssert;
```

Comment:
I have not found a better way how it would be possible to pass values ​​for validation